### PR TITLE
Dashboards: allow changing date ranges on /instance/status dashboards

### PR DESCRIPTION
**Pull request by macobo**
- Show more things when embedding dashboard for internal purposes
- Show refresh button for /instance/status dashboard
- Allow logged in users to change date range in /instance/status dashboard

image

Related ticket: https://github.com/PostHog/vpc/issues/45

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

File: frontend/src/types.ts [hunk 1]
```diff
@@ -682,6 +682,7 @@ export enum DashboardMode { // Default mode is null
     Fullscreen = 'fullscreen', // When the dashboard is on full screen (presentation) mode
     Sharing = 'sharing', // When the sharing configuration is opened
     Public = 'public', // When viewing the dashboard publicly via a shareToken
+    Internal = 'internal', // When embedded into another page (e.g. /instance/status)
```
Comment (paolodamico) [inline frontend/src/types.ts:685]: Nit: description seems a bit confusing; like embedding dashboards is a common occurrence, though I think we truly only support the instance/status page, right?

File: frontend/src/scenes/dashboard/DashboardItems.tsx [hunk 2]
```diff
@@ -99,7 +99,7 @@ export function DashboardItems({ inSharedMode }: { inSharedMode: boolean }): JSX
                         }
                         updateItemColor={updateItemColor}
                         isDraggingRef={isDragging}
-                        inSharedMode={inSharedMode}
+                        dashboardMode={dashboardMode}
                         isOnEditMode={dashboardMode === DashboardMode.Edit}
```
Comment (paolodamico) [inline frontend/src/scenes/dashboard/DashboardItems.tsx:103]: Now that we're passing the `dashboardMode` fully downstream, seems like we should refactor this too.

**Review by paolodamico — APPROVED** [review]
lgtm!

File: frontend/src/scenes/dashboard/DashboardItems.tsx (hunk 2 above)
Comment (macobo) [reply frontend/src/scenes/dashboard/DashboardItems.tsx:103]: :+1: Agreed, will open a follow-up

**Pull request merged**